config/routes.rb

```diff
@@ -3,4 +3,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  resources :dishes, only: [:show]
+  post "/dishes/:id", to: "dishes#create"
+
+  resources :chefs, only: [:show]
+
+  get "/chefs/:id/igredients", to: "chefs/ingredients#index"
```

Comment:
Again, it's not required to use advanced routing on projects, but what you could do for this handrolled route is:
```
resources :chefs, only: :show do
    resources :ingredients, only: :index, controller: "chefs/ingredients#index"
end
```

If handrolling feels easier on an IC, that is 100% okay!